tests/test_rpc.py

```diff
@@ -261,7 +299,9 @@ def {}():
 
         test_case.assert_attr(name=test_name)
         assert not property_present(test_case, 'test_id')
+        assert not property_present(test_case, 'jira')
 
+    @pytest.mark.skipif('SKIP_LONG_RUNNING_TESTS' in os.environ, reason='Impatient developer is impatient')
```

Comment:
hehe

Reply:
![https://media.giphy.com/media/NyjObFYNmwEM/giphy.gif](Nobody got time for that)